### ISSUE TITLE
E24 - Subscript not working

### ISSUE THREAD

**Issue by ellemindwave** [LDMW/cms#774]
This is what is passed in asset
```
[Is he/Are you] taking any <b>prescription medicines</b>?
<span class="e24subtext">This includes any medication or supplements by mouth, inhaled, by skin patch, injected, inserted or applied. Medication may be used either daily, like blood pressure medicine, or intermittently, like migraine medicine.</span>
```
This is in apps (correctly displaying):

![screen shot 2017-11-08 at 16 47 36](https://user-images.githubusercontent.com/26304634/32561599-8bfc0124-c4a4-11e7-90c1-3fdcb39ad4d4.png)

This is in ldmh all questions are in bold

![screen shot 2017-11-08 at 16 48 53](https://user-images.githubusercontent.com/26304634/32561668-baab91d8-c4a4-11e7-90e4-43f3f84f44bf.png)

**Comment by katbow** [comment]
@ellemindwave how is this part meant to be displayed?

![image](https://user-images.githubusercontent.com/1287388/32780883-cf3b01de-c93a-11e7-82ac-1803197fc729.png)

Is only part of that meant to be bold or something?

I found that a `font-weight:600` is applied to all `li` elements across the site. If I turn that off, it looks like this:

![image](https://user-images.githubusercontent.com/1287388/32781104-a641fc00-c93b-11e7-8d27-6aaf9c42bf86.png)

**Comment by katbow** [comment]
@ellemindwave , nevermind actually, that part appears to be part of #768 & the question I asked there! Will focus on the question part for this issue.

**Comment by katbow** [comment]
The second part of this was fixed with #767 by adding the e24subtext class.

The question itself is because this is an h3 tag, which is styled across the site as:

```
h3 {
  font-family: 'Montserrat', sans-serif;
  letter-spacing: 0.5pt;
  font-weight: 600;
  font-size: 16px;
  line-height: 22px;
}
```

The `font-weight: 600;` makes the entire question bold by default. I'm adding in more specific css to remove this when in the form. However I'm not sure if this will still allow bold at all. I'm checking that out now.

The html for the question is:
```
<h3>
      <div class="rich-text">Question goes here.</div>
</h3>
```

I'm checking what rich text does and if I need to make changes for this.

**Comment by katbow** [comment]
The h3 was part of our code actually so I have switched it to a `div` to match the styling of a single question.

I found some questions that contain bold, and it now displays like this:
![image](https://user-images.githubusercontent.com/1287388/32791469-07b523a0-c959-11e7-83ae-260e865ffa33.png)
